### PR TITLE
linux hack for gatttool command invocation

### PR DESCRIPTION
**Pull request by claymodel**
work around for the 
non standard BLE CCCD (not read able CCCD)

### CODE REVIEW

File: lib/hci-socket/bindings.js [hunk 1]
```diff
@@ -392,7 +392,10 @@ NobleBindings.prototype.notify = function(peripheralUuid, serviceUuid, character
   var handle = this._handles[peripheralUuid];
   var gatt = this._gatts[handle];
 
-  if (gatt) {
+  if (handle == 0x000f) {
```
Comment (sandeepmistry) [inline lib/hci-socket/bindings.js]: I'm not too keen on merging this, this change is super specific to your device, other devices could use handle 0x0f and not have this issue.

**Comment by sandeepmistry** [comment]
@claymodel thanks for submitting this, unfortunately based on my comment above I'm not keen on merging this.

What's the issue you are facing? What BLE peripheral has the issue? I'd much prefer to fix the underlying issue instead of the device specific workaround you proposed.

**Comment by jrobeson** [comment]
I think it'd be fair to close this and let them reopen it if they want to elaborate.

**Comment by sandeepmistry** [comment]
@jrobeson good suggestion!

@claymodel please see @jrobeson's comment above.

**Pull request closed without merging**